src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -260,6 +260,9 @@
     <Compile Include="$(CommonPath)\Interop\Unix\System.Security.Cryptography.Native\Interop.Crypto.cs">
       <Link>Common\Interop\Unix\System.Security.Cryptography.Native\Interop.Crypto.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Security.Cryptography.Native\Interop.X509Name.cs">
```

Comment:
I guess now it says you've moved it (it likely got added by someone else in the meantime due to the ongoing p/invoke changes).

Please rebase onto master and squash your changes,